### ISSUE TITLE
[Critical] Function cannot be performed by keyboard alone

### ISSUE THREAD

**Issue by ntallman** [APTrust/dart#585]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87606 |
| **Checkpoint** | 2.1 Keyboard Accessible › Keyboard Navigation (2.1.1.a) |
| **Impact** | Critical |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 01 Settings Workflow |
| **URL** | <https://aptrust.org/documentation-page/dart-digital-archivists-resource-tool/> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Date Reported** | 05/06/26 at 4:45 pm |
| **Issue Status** | Open |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | kim.casey@ablr360.com |

## Description

There is no way to perform the function using only the keyboard on the same screen or on a qualifying conforming alternate version.
In Settings menu, select BagIt Profiles, then New, then accept the default of AP Trust, then select Create. Expand the Tag Files, select aptrust-info.txt. On that page, all of the items in the Tag Name table cannot be selected using the keyboard. The focus goes from New Tag (before table) to Delete Profile (after table).  See Settings User Experience video at 10:55.

## Recommendation

Rule : 
Links, buttons, and interactive controls MUST be keyboard-focusable and operable.

How To Fix: 
If the control is doing a "button action" as opposed to navigating to another location, fix this issue by using ONE of the following techniques:

1. Use a native HTML <button> element. A <button> element comes with keyboard accessibility baked in.

<button>Quick view</button>

2. Ensure the control is:
a. Keyboard focusable (this may involve adding tabindex="0" to make it focusable) AND
b. Keyboard operable using the Enter and Spacebar keys in addition to mouse click (this may involve adding JavaScript event handlers for those keys).

NOTE: If you do not use a <button>, you will also need to add role="button" to ensure the button has a role to satisfy 4.1.2 Name, Role, Value, and a focus indicator will be needed to satisfy 2.4.7 Focus Visible.

<div tabindex="0" role="button">Quick view</div>

Background: 
Some people cannot use a mouse due to vision or motor disabilities. Content that can be operated with a mouse must also be made operable with a keyboard. When content is operable through a keyboard, it becomes operable by a variety of assistive technologies such as speech input software, sip-and-puff software, on-screen keyboards, scanning software, and alternate keyboards.

## Screenshots

![01_keyboard1](https://raw.githubusercontent.com/APTrust/dart/master/docs/accessibility-audit/screenshots/87606/01_keyboard1.png)
